### DISCUSSION TITLE
500 SERVER ERROR

### DISCUSSION THREAD

**Discussion by mahseema** [yhype/yhype-feedback#20] [General]
I couldn't Login in Y Hype

**Comment by antonkomarev** [comment]
@mahseema should work now

**Comment by mahseema** [comment]
Oh no. It steel gives me 500 error.

**Reply by antonkomarev** [reply]
I see last issue was `[2023-08-31 09:03:17 UTC]`, after then no issues. Thats strange. Could you write on [Discord](https://discord.gg/nFBSdHFyqm) tomorrow? I will try to debug in realtime what's happening.

**Reply by antonkomarev** [reply]
I see that your profile created and data is collected.

**Comment by cubicbyte** [comment]
Still happens when creating an account

**Reply by antonkomarev** [reply]
@cubicbyte try again please

**Reply by cubicbyte** [reply]
Now it works 👍

**Comment by valerionew** [comment]
Hey! I'm still getting the 500 server error message

**Reply by antonkomarev** [reply]
@valerionew Fixed it

**Reply by valerionew** [reply]
Thanks!

**Comment by antonkomarev** [comment]
We've done some tools for periodic redis cleanup. Hope it will help us to stay alive without issues.

**Reply by antonkomarev** [reply]
It helped but not much, we are still going 500 every morning :( Going to solve this in near future.

**Comment by antonkomarev** [comment]
We found 2 issues and resolved them. Should not return 500 now. 2 days without issues. Thanks for reporting!
